addons/point_of_sale/static/src/app/customer_display/customer_display_adapter.js

```diff
@@ -30,6 +30,7 @@ export class CustomerDisplayPosAdapter {
     }
 
     formatOrderData(order) {
+        this.order = order;
```

Comment:
Maybe you can just store the currency instead of the order object, as I cannot see any use case for storing the whole order object 
```suggestion
        this.currency = order.currency;
```